packages/marshal/docs/marshal-vs-store-level.md

```diff
@@ -0,0 +1,95 @@
+
+The **primitive styles** identify the *passable primitives*. The passable primitives have the expected mapping down to the JavaScript primitives. However, the passable primitives do not include JavaScript's `-0` or anonymous symbols. Those JavaScript primitives that are included round-trip through the protocol.
+
+The **data styles** tag data-like containers. If a data-like container contains only data, then it is only data, passing the `isOnlyData` type test.
+  * A *copyArray* is a simple JavaScript array containing only passables.
+  * A *copyRecord* is a simple JavaScript record-like object, with only string-named own data properties containing only passables.
+  * A *copyTagged* is the extensibility point used by the Store level to implement *copySets* and *copyMaps*.
+
+The **structure styles** introduce the *remotable*, which is a potentially remote behavioral object with a unique unforgeable identity and a primary location. At that primary location is the remotable's *Primary Remotable*, often referred to as a *far object*. At all other locations, the remotable is represented by a *remote*, implemented as a JavaScript handled-promise *presence*. Any message sent to a remotable is delivered to its primary remotable, invoking one of its methods. A remotable is a *structure*. A data-like container containing only structures, i.e., data and remotables, is itself a structure.
+
+Each remaining **passable** is either
+  * An *error* carrying diagnostic information.
+  * A *promise*, providing the eventual distributed access that JavaScript's promises were designed to provide.
+  * A *metaTagged*, like the copyTagged, is the extension point used by the Store level to implement
+ *patternNodes*. The distinction is that a metaTagged is necessarily *not* a structure or a key. Thus, a patternNode, encoded into a metaTagged, can be applied to a structure or key specimen, knowing that the specimen cannot itself contain a patternNode. This avoids a tricky source of ambiguity and confusion.
+
+**`sameStructure(s: Structure, t: Structure) => boolean`**<br>
+ implements the most precise equivalence class meaningful at the Marshal layer. However, it is narrow, admitting only structures.
+
+**`comparePassables(left: Passable, right :Passable) => -1 | 0 | 1`**<br>
+implements a full order over *all* passables, suitable for sorting with `Array.prototype.sort`. However, it is a very imprecise full order. For example, for any two remotables `r` and `q`, `comparePassables(r,q) === 0`. All remotables are therefore equivalent in this order. Likewise for all errors, all promises, and all metaTaggeds. Thus, `comparePassable`'s zero answer provides an equivalence class over all passables, but an imprecise one.
+
+ ## The Store level organized by `keyStyleOf`
+
+**`keyStyleOf(p: Passable) => KeyStyle`**<br>
+ where `KeyStyle` is an enumeration quite similar to `PassStyle`
+
+`KeyStyle`:
```

Comment:
Instead of re-creating a similar hierarchy, consider explaining just the differences:

* The following are identical: primitive styles (`"undefined"`, `"null"`, `"boolean"`, `"bigint"`, `"number"`, `"string"`, `"symbol"`), `"copyArray"`, `"copyRecord"`, `"remotable"`.
* The following are disallowed and will throw if passed to `keyStyleOf()`: errors, promises, and metaTagged.
* Data with a `passStyle` of `"copyTagged"` must have a valid representation, and will have a `keyStyle` of `"copySet"` or `"copyMap"` depending on that representation.